src/config_ctrl.js

```diff
@@ -8,10 +8,26 @@ export class OCIConfigCtrl {
   /** @ngInject */
   constructor ($scope, backendSrv) {
     this.backendSrv = backendSrv
-    this.tenancyOCID = this.current.jsonData.tenancyOCID
-    this.defaultRegion = this.current.jsonData.defaultRegion
     this.environment = this.current.jsonData.environment      
     this.tenancymode = this.current.jsonData.tenancymode
+    this.defaultRegion = this.current.jsonData.defaultRegion
+    this.region0 = this.current.jsonData.region0
```

Comment:
can we not have a map of objects/structures. It will be more cleaner IMO and also you can't still restrict it to 5 entries

Reply:
Grafana requires a discrete amount of parameters, that the reason why this is not a dynamic map.